SPECS/networkd-dispatcher/networkd-dispatcher.spec

```diff
@@ -0,0 +1,62 @@
+%global debug_package %{nil}
+
+Summary:        A dispatcher daemon for systemd-networkd events
+Name:           networkd-dispatcher
+Version:        2.2.4
+Release:        1%{?dist}
+License:        GPLv3+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://gitlab.com/craftyguy/networkd-dispatcher
+Source0:        https://gitlab.com/craftyguy/%{name}/-/archive/%{version}/%{name}-%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  asciidoc
+Requires:       python3
+Requires:       dbus-python
```

Comment:
python3-gobject & python3-dbus

Reply:
Corrected the packages to the names you mentioned